frontend/apps/marketing/src/components/contentful/link/LinkContentfulDefinition.ts

```diff
@@ -43,6 +43,7 @@ export const LinkContentfulComponentDefinition: ComponentDefinition = {
     href: {
       displayName: 'Link URL',
       type: 'Text',
+      defaultValue: 'https://code.org',
```

Comment:
Thinking the default shouldn't be here as it would possibly result in some links pointing to code.org accidentally (would prefer the editor to have to enter it in). Happy to have this as a follow up item